modular_skyrat/modules/modular_items/lewd_items/code/lewd_structures/lustwish.dm

```diff
@@ -75,14 +75,11 @@
 					//hands
 					/obj/item/clothing/gloves/ball_mittens = 8,
 					/obj/item/clothing/gloves/latex_gloves = 8,
-					/obj/item/clothing/gloves/evening = 5,
 
 					//legs
 					/obj/item/clothing/shoes/latex_socks = 8,
 					/obj/item/clothing/shoes/latexheels = 4,
 					/obj/item/clothing/shoes/dominaheels = 4,
-					/obj/item/clothing/shoes/jackboots/thigh = 3,
-					/obj/item/clothing/shoes/jackboots/knee = 3,
```

Comment:
I just feel like there's a lot of removals in this PR that are both not documented, and are done with no alternatives available to the removed content. If you want sprite consistency, then bring those up to speed, not everything needs to be entirely new and chuck full of mechanics. People want those, if you want to keep visual consistency, then just resprite them. Content for another PR anyway.